tcp-ballerina/tests/client_tests.bal

```diff
@@ -75,6 +75,43 @@ function testServerAlreadyClosed() returns  @tainted error? {
     check socketClient->close();
 }
 
+@test:Config {
+    dependsOn: [testServerAlreadyClosed]
+}
+function testReadBytesAsStream() returns  @tainted error? {
+    Client socketClient = check new ("localhost", PORT4);
+
+    Error|(stream<readonly & byte[]>) res = socketClient->readBytesAsStream();
+    if (res is stream<readonly & byte[]>) {
+        error? e = res.forEach(function (readonly & byte[] bytes) {
+                io:println(getString(bytes));
```

Comment:
Shouldn’t we need to assert this?

Reply:
added `test:assertFail(msg = e.message());` in if